<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="22" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:25:05 INFO  ==============================================================
18:25:05 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
18:25:05 INFO  Dir:   plugin_apama/
18:25:05 INFO  Title: Validate apama plugin install
18:25:05 INFO  ==============================================================
18:25:05 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:25:05 WARN  Aborted test due to SKIPPED outcome
18:25:05 INFO  
18:25:05 INFO  Test duration: 0.05 secs
18:25:05 INFO  Test final outcome:  SKIPPED
18:25:05 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:25:05 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:25:05 INFO  ==============================================================
18:25:05 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
18:25:05 INFO  Dir:   plugin_apama/
18:25:05 INFO  Title: Validate mon file support of apama plugin
18:25:05 INFO  ==============================================================
18:25:05 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:25:05 WARN  Aborted test due to SKIPPED outcome
18:25:05 INFO  
18:25:05 INFO  Test duration: 0.02 secs
18:25:05 INFO  Test final outcome:  SKIPPED
18:25:05 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:25:05 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:25:05 INFO  ==============================================================
18:25:05 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
18:25:05 INFO  Dir:   plugin_apama/
18:25:05 INFO  Title: Validate apama plugin remove
18:25:05 INFO  ==============================================================
18:25:05 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:25:05 WARN  Aborted test due to SKIPPED outcome
18:25:05 INFO  
18:25:05 INFO  Test duration: 0.05 secs
18:25:05 INFO  Test final outcome:  SKIPPED
18:25:05 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:25:05 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
18:25:06 INFO  Dir:   plugin_apama/
18:25:06 INFO  Title: Validate apama plugin updating one project with another
18:25:06 INFO  ==============================================================
18:25:06 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.06 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>19:16:31 INFO  ==============================================================
19:16:31 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
19:16:31 INFO  Dir:   plugin_apama/
19:16:31 INFO  Title: Validate apama plugin updating one project with another
19:16:31 INFO  ==============================================================
19:16:31 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
19:16:31 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
19:16:32 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
19:16:32 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
19:16:33 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
19:16:34 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
19:16:34 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
19:16:35 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
19:16:35 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
19:16:35 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:16:35 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

19:16:36 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
19:16:36 WARN  Aborted test due to BLOCKED outcome
19:16:36 INFO  
19:16:36 INFO  cleanup:
19:16:36 INFO  Removing apama project
19:16:51 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
19:16:52 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
19:16:52 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
19:16:52 INFO  
19:16:52 INFO  Test duration: 21.66 secs
19:16:52 INFO  Test final outcome:  BLOCKED
19:16:52 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
19:16:52 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>19:21:19 INFO  ==============================================================
19:21:19 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
19:21:19 INFO  Dir:   plugin_apama/
19:21:19 INFO  Title: Validate apama plugin updating one project with another
19:21:19 INFO  ==============================================================
19:21:19 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
19:21:19 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
19:21:20 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
19:21:20 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
19:21:21 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
19:21:22 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
19:21:22 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
19:21:23 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
19:21:23 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
19:21:23 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
19:21:23 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

19:21:24 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
19:21:24 WARN  Aborted test due to BLOCKED outcome
19:21:24 INFO  
19:21:24 INFO  cleanup:
19:21:24 INFO  Removing apama project
19:21:39 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
19:21:40 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
19:21:40 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
19:21:40 INFO  
19:21:40 INFO  Test duration: 21.65 secs
19:21:40 INFO  Test final outcome:  BLOCKED
19:21:40 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
19:21:40 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:48 INFO  ==============================================================
18:34:48 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
18:34:48 INFO  Dir:   software_management_end_to_end/
18:34:48 INFO  Title: Install package to directory specified in tedge config set tmp.path
18:34:48 INFO  ==============================================================
18:34:48 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:48 WARN  Aborted test due to SKIPPED outcome
18:34:48 INFO  
18:34:48 INFO  Test duration: 0.12 secs
18:34:48 INFO  Test final outcome:  SKIPPED
18:34:48 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:48 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>18:58:29 INFO  ==============================================================
18:58:29 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
18:58:29 INFO  Dir:   software_management_end_to_end/
18:58:29 INFO  Title: Install package to directory specified in tedge config set tmp.path
18:58:29 INFO  ==============================================================
18:58:29 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
18:58:29 INFO  EnvironmentC8y Setup
18:58:30 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
18:58:44 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:58:46 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
18:58:47 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
18:58:48 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
18:58:50 INFO  State of current operation: SUCCESSFUL
18:58:50 INFO  Assert that (False == value) with value=False ... passed
18:58:51 INFO  Executed sudo, exit status 0
18:58:51 INFO  Executed sudo, exit status 0
18:58:52 INFO  Executed sudo, exit status 0
18:58:56 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:58:56 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
18:58:56 INFO  Delaying execution by 8 seconds
18:59:18 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
18:59:18 INFO  Response status: 201
18:59:18 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-03T18:59:18.321Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-03T18:59:18.308Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334205",
    "id": "18334205",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
18:59:18 INFO  Started operation: {'delivery': {'time': '2022-05-03T18:59:18.321Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-03T18:59:18.308Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334205', 'id': '18334205', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
18:59:18 WARN  , retrying in 0.5 seconds...
18:59:18 WARN  , retrying in 0.5 seconds...
18:59:19 WARN  , retrying in 0.5 seconds...
18:59:19 WARN  , retrying in 0.5 seconds...
18:59:20 WARN  , retrying in 0.5 seconds...
18:59:20 WARN  , retrying in 0.5 seconds...
18:59:21 WARN  , retrying in 0.5 seconds...
18:59:21 WARN  , retrying in 0.5 seconds...
18:59:22 WARN  , retrying in 0.5 seconds...
18:59:22 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
18:59:23 WARN  AssertionError: ... blocked 
18:59:23 INFO  
18:59:23 INFO  cleanup:
18:59:24 INFO  Executed sudo, exit status 0
18:59:24 INFO  Executed sudo, exit status 0
18:59:29 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:59:29 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
18:59:29 INFO  Delaying execution by 8 seconds
18:59:51 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:59:55 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:59:55 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
18:59:55 INFO  
18:59:55 INFO  Test duration: 86.69 secs
18:59:55 INFO  Test final outcome:  BLOCKED
18:59:55 INFO  Test outcome reason: AssertionError:
18:59:55 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>19:21:40 INFO  ==============================================================
19:21:40 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
19:21:40 INFO  Dir:   software_management_end_to_end/
19:21:40 INFO  Title: Install package to directory specified in tedge config set tmp.path
19:21:40 INFO  ==============================================================
19:21:42 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:21:42 INFO  EnvironmentC8y Setup
19:21:42 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:21:57 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:21:57 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:21:58 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:21:58 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:22:00 INFO  State of current operation: FAILED
19:22:01 INFO  Assert that (False == value) with value=False ... passed
19:22:01 INFO  Executed sudo, exit status 0
19:22:02 INFO  Executed sudo, exit status 0
19:22:02 INFO  Executed sudo, exit status 0
19:22:06 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:22:06 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:22:06 INFO  Delaying execution by 8 seconds
19:22:28 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:22:28 INFO  Response status: 201
19:22:28 INFO  Response to action: {
    "creationTime": "2022-05-03T19:22:28.729Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334342",
    "id": "18334342",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
19:22:28 INFO  Started operation: {'creationTime': '2022-05-03T19:22:28.729Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334342', 'id': '18334342', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
19:22:28 WARN  , retrying in 0.5 seconds...
19:22:29 WARN  , retrying in 0.5 seconds...
19:22:29 WARN  , retrying in 0.5 seconds...
19:22:30 WARN  , retrying in 0.5 seconds...
19:22:30 WARN  , retrying in 0.5 seconds...
19:22:31 WARN  , retrying in 0.5 seconds...
19:22:32 WARN  , retrying in 0.5 seconds...
19:22:32 WARN  , retrying in 0.5 seconds...
19:22:33 WARN  , retrying in 0.5 seconds...
19:22:33 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
19:22:33 WARN  AssertionError: ... blocked 
19:22:33 INFO  
19:22:33 INFO  cleanup:
19:22:34 INFO  Executed sudo, exit status 0
19:22:34 INFO  Executed sudo, exit status 0
19:22:39 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:22:39 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
19:22:39 INFO  Delaying execution by 8 seconds
19:23:01 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:23:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:23:05 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:23:05 INFO  
19:23:05 INFO  Test duration: 84.85 secs
19:23:05 INFO  Test final outcome:  BLOCKED
19:23:05 INFO  Test outcome reason: AssertionError:
19:23:05 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:48 INFO  ==============================================================
18:34:48 INFO  Id:    SoftwareManagement.sm_apt_install_fail
18:34:48 INFO  Dir:   software_management_end_to_end/
18:34:48 INFO  Title: Validate end to end behaviour for a failing installation
18:34:48 INFO  ==============================================================
18:34:48 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:48 WARN  Aborted test due to SKIPPED outcome
18:34:48 INFO  
18:34:48 INFO  Test duration: 0.02 secs
18:34:48 INFO  Test final outcome:  SKIPPED
18:34:48 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:48 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:48 INFO  ==============================================================
18:34:48 INFO  Id:    SoftwareManagement.sm_apt_install_remove
18:34:48 INFO  Dir:   software_management_end_to_end/
18:34:48 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
18:34:48 INFO  ==============================================================
18:34:48 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:48 WARN  Aborted test due to SKIPPED outcome
18:34:48 INFO  
18:34:48 INFO  Test duration: 0.02 secs
18:34:48 INFO  Test final outcome:  SKIPPED
18:34:48 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:48 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:48 INFO  ==============================================================
18:34:48 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
18:34:48 INFO  Dir:   software_management_end_to_end/
18:34:48 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
18:34:48 INFO  ==============================================================
18:34:48 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:48 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.02 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.02 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>19:06:44 INFO  ==============================================================
19:06:44 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
19:06:44 INFO  Dir:   software_management_end_to_end/
19:06:44 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
19:06:44 INFO  ==============================================================
19:06:44 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:06:44 INFO  EnvironmentC8y Setup
19:06:44 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:06:44 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:06:44 INFO  Delaying execution by 6 seconds
19:07:05 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:07:05 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:07:06 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:07:06 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:07:08 INFO  State of current operation: SUCCESSFUL
19:07:09 INFO  Assert that (False == value) with value=False ... passed
19:07:09 INFO  Assert that (False == value) with value=False ... passed
19:07:09 INFO  Assert that (False == value) with value=False ... passed
19:07:09 INFO  Package robotfindskitten is installed
19:07:09 WARN  Assert that (False == value) with value=True ... failed 
19:07:10 INFO  Response status: 201
19:07:10 INFO  Response to action: {
    "creationTime": "2022-05-03T19:07:09.979Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334254",
    "id": "18334254",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:07:10 INFO  Started operation: {'creationTime': '2022-05-03T19:07:09.979Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334254', 'id': '18334254', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:07:12 INFO  State of operation 18334254 : PENDING
19:07:14 INFO  State of operation 18334254 : EXECUTING
19:07:16 INFO  State of operation 18334254 : EXECUTING
19:07:19 INFO  State of operation 18334254 : EXECUTING
19:07:21 INFO  State of operation 18334254 : EXECUTING
19:07:23 INFO  State of operation 18334254 : EXECUTING
19:07:26 INFO  State of operation 18334254 : EXECUTING
19:07:28 INFO  State of operation 18334254 : EXECUTING
19:07:31 INFO  State of operation 18334254 : EXECUTING
19:07:33 INFO  State of operation 18334254 : EXECUTING
19:07:35 INFO  State of operation 18334254 : EXECUTING
19:07:38 INFO  State of operation 18334254 : EXECUTING
19:07:40 INFO  State of operation 18334254 : EXECUTING
19:07:42 INFO  State of operation 18334254 : EXECUTING
19:07:44 INFO  State of operation 18334254 : EXECUTING
19:07:47 INFO  State of operation 18334254 : EXECUTING
19:07:49 INFO  State of operation 18334254 : EXECUTING
19:07:52 INFO  State of operation 18334254 : EXECUTING
19:07:54 INFO  State of operation 18334254 : EXECUTING
19:07:56 INFO  State of operation 18334254 : EXECUTING
19:07:58 INFO  State of operation 18334254 : EXECUTING
19:08:01 INFO  State of operation 18334254 : EXECUTING
19:08:03 INFO  State of operation 18334254 : EXECUTING
19:08:06 INFO  State of operation 18334254 : EXECUTING
19:08:08 INFO  State of operation 18334254 : EXECUTING
19:08:10 INFO  State of operation 18334254 : EXECUTING
19:08:13 INFO  State of operation 18334254 : EXECUTING
19:08:15 INFO  State of operation 18334254 : EXECUTING
19:08:17 INFO  State of operation 18334254 : EXECUTING
19:08:19 INFO  State of operation 18334254 : EXECUTING
19:08:22 INFO  State of operation 18334254 : EXECUTING
19:08:24 INFO  State of operation 18334254 : EXECUTING
19:08:26 INFO  State of operation 18334254 : EXECUTING
19:08:28 INFO  State of operation 18334254 : EXECUTING
19:08:31 INFO  State of operation 18334254 : SUCCESSFUL
19:08:31 INFO  Assert that (False == value) with value=False ... passed
19:08:31 INFO  Package asciijump is installed
19:08:31 INFO  Assert that (True == value) with value=True ... passed
19:08:31 INFO  Package squirrel3 is installed
19:08:31 INFO  Assert that (True == value) with value=True ... passed
19:08:31 INFO  Package robotfindskitten is installed
19:08:31 INFO  Assert that (True == value) with value=True ... passed
19:08:32 INFO  Response status: 201
19:08:32 INFO  Response to action: {
    "creationTime": "2022-05-03T19:08:32.151Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334264",
    "id": "18334264",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:08:32 INFO  Started operation: {'creationTime': '2022-05-03T19:08:32.151Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334264', 'id': '18334264', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:08:34 INFO  State of operation 18334264 : PENDING
19:08:36 INFO  State of operation 18334264 : EXECUTING
19:08:38 INFO  State of operation 18334264 : EXECUTING
19:08:41 INFO  State of operation 18334264 : EXECUTING
19:08:43 INFO  State of operation 18334264 : EXECUTING
19:08:45 INFO  State of operation 18334264 : EXECUTING
19:08:48 INFO  State of operation 18334264 : EXECUTING
19:08:51 INFO  State of operation 18334264 : EXECUTING
19:08:53 INFO  State of operation 18334264 : EXECUTING
19:08:55 INFO  State of operation 18334264 : EXECUTING
19:08:57 INFO  State of operation 18334264 : EXECUTING
19:09:00 INFO  State of operation 18334264 : EXECUTING
19:09:02 INFO  State of operation 18334264 : EXECUTING
19:09:04 INFO  State of operation 18334264 : EXECUTING
19:09:06 INFO  State of operation 18334264 : EXECUTING
19:09:09 INFO  State of operation 18334264 : EXECUTING
19:09:11 INFO  State of operation 18334264 : EXECUTING
19:09:14 INFO  State of operation 18334264 : EXECUTING
19:09:16 INFO  State of operation 18334264 : EXECUTING
19:09:18 INFO  State of operation 18334264 : EXECUTING
19:09:21 INFO  State of operation 18334264 : EXECUTING
19:09:23 INFO  State of operation 18334264 : EXECUTING
19:09:25 INFO  State of operation 18334264 : EXECUTING
19:09:27 INFO  State of operation 18334264 : EXECUTING
19:09:30 INFO  State of operation 18334264 : EXECUTING
19:09:33 INFO  State of operation 18334264 : EXECUTING
19:09:35 INFO  State of operation 18334264 : EXECUTING
19:09:37 INFO  State of operation 18334264 : SUCCESSFUL
19:09:37 INFO  Assert that (False == value) with value=False ... passed
19:09:38 INFO  Assert that (False == value) with value=False ... passed
19:09:38 INFO  Assert that (False == value) with value=False ... passed
19:09:38 INFO  Assert that (False == value) with value=False ... passed
19:09:38 INFO  
19:09:38 INFO  cleanup:
19:09:42 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:09:43 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:09:43 INFO  
19:09:43 INFO  Test duration: 179.01 secs
19:09:43 INFO  Test final outcome:  FAILED
19:09:43 INFO  Test outcome reason: Assert that (False == value) with value=True
19:09:43 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>19:29:34 INFO  ==============================================================
19:29:34 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
19:29:34 INFO  Dir:   software_management_end_to_end/
19:29:34 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
19:29:34 INFO  ==============================================================
19:29:34 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:29:34 INFO  EnvironmentC8y Setup
19:29:34 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:29:34 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:29:34 INFO  Delaying execution by 6 seconds
19:29:54 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:29:55 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:29:55 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:29:56 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:29:58 INFO  State of current operation: SUCCESSFUL
19:29:58 INFO  Assert that (False == value) with value=False ... passed
19:29:59 INFO  Package asciijump is installed
19:29:59 WARN  Assert that (False == value) with value=True ... failed 
19:29:59 INFO  Package squirrel3 is installed
19:29:59 WARN  Assert that (False == value) with value=True ... failed 
19:29:59 INFO  Package robotfindskitten is installed
19:29:59 WARN  Assert that (False == value) with value=True ... failed 
19:30:00 INFO  Response status: 201
19:30:00 INFO  Response to action: {
    "creationTime": "2022-05-03T19:30:00.040Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334419",
    "id": "18334419",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:30:00 INFO  Started operation: {'creationTime': '2022-05-03T19:30:00.040Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334419', 'id': '18334419', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:30:02 INFO  State of operation 18334419 : PENDING
19:30:04 INFO  State of operation 18334419 : EXECUTING
19:30:06 INFO  State of operation 18334419 : EXECUTING
19:30:09 INFO  State of operation 18334419 : EXECUTING
19:30:11 INFO  State of operation 18334419 : EXECUTING
19:30:13 INFO  State of operation 18334419 : EXECUTING
19:30:16 INFO  State of operation 18334419 : EXECUTING
19:30:18 INFO  State of operation 18334419 : EXECUTING
19:30:21 INFO  State of operation 18334419 : EXECUTING
19:30:23 INFO  State of operation 18334419 : EXECUTING
19:30:25 INFO  State of operation 18334419 : EXECUTING
19:30:28 INFO  State of operation 18334419 : EXECUTING
19:30:30 INFO  State of operation 18334419 : EXECUTING
19:30:32 INFO  State of operation 18334419 : EXECUTING
19:30:34 INFO  State of operation 18334419 : EXECUTING
19:30:37 INFO  State of operation 18334419 : SUCCESSFUL
19:30:37 INFO  Assert that (False == value) with value=False ... passed
19:30:37 INFO  Package asciijump is installed
19:30:37 INFO  Assert that (True == value) with value=True ... passed
19:30:37 INFO  Package squirrel3 is installed
19:30:37 INFO  Assert that (True == value) with value=True ... passed
19:30:37 INFO  Package robotfindskitten is installed
19:30:37 INFO  Assert that (True == value) with value=True ... passed
19:30:38 INFO  Response status: 201
19:30:38 INFO  Response to action: {
    "creationTime": "2022-05-03T19:30:38.156Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334433",
    "id": "18334433",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:30:38 INFO  Started operation: {'creationTime': '2022-05-03T19:30:38.156Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334433', 'id': '18334433', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
19:30:40 INFO  State of operation 18334433 : PENDING
19:30:42 INFO  State of operation 18334433 : EXECUTING
19:30:44 INFO  State of operation 18334433 : EXECUTING
19:30:47 INFO  State of operation 18334433 : EXECUTING
19:30:49 INFO  State of operation 18334433 : EXECUTING
19:30:51 INFO  State of operation 18334433 : EXECUTING
19:30:54 INFO  State of operation 18334433 : EXECUTING
19:30:57 INFO  State of operation 18334433 : EXECUTING
19:30:59 INFO  State of operation 18334433 : EXECUTING
19:31:01 INFO  State of operation 18334433 : EXECUTING
19:31:04 INFO  State of operation 18334433 : EXECUTING
19:31:06 INFO  State of operation 18334433 : EXECUTING
19:31:08 INFO  State of operation 18334433 : EXECUTING
19:31:10 INFO  State of operation 18334433 : EXECUTING
19:31:13 INFO  State of operation 18334433 : EXECUTING
19:31:15 INFO  State of operation 18334433 : EXECUTING
19:31:17 INFO  State of operation 18334433 : EXECUTING
19:31:20 INFO  State of operation 18334433 : EXECUTING
19:31:22 INFO  State of operation 18334433 : EXECUTING
19:31:25 INFO  State of operation 18334433 : EXECUTING
19:31:27 INFO  State of operation 18334433 : EXECUTING
19:31:29 INFO  State of operation 18334433 : EXECUTING
19:31:31 INFO  State of operation 18334433 : EXECUTING
19:31:33 INFO  State of operation 18334433 : EXECUTING
19:31:36 INFO  State of operation 18334433 : EXECUTING
19:31:38 INFO  State of operation 18334433 : EXECUTING
19:31:41 INFO  State of operation 18334433 : EXECUTING
19:31:43 INFO  State of operation 18334433 : EXECUTING
19:31:45 INFO  State of operation 18334433 : EXECUTING
19:31:47 INFO  State of operation 18334433 : SUCCESSFUL
19:31:48 INFO  Assert that (False == value) with value=False ... passed
19:31:48 INFO  Assert that (False == value) with value=False ... passed
19:31:48 INFO  Assert that (False == value) with value=False ... passed
19:31:48 INFO  Assert that (False == value) with value=False ... passed
19:31:48 INFO  
19:31:48 INFO  cleanup:
19:31:52 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:31:53 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:31:53 INFO  
19:31:53 INFO  Test duration: 139.55 secs
19:31:53 INFO  Test final outcome:  FAILED
19:31:53 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
19:31:53 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.03 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>19:09:43 INFO  ==============================================================
19:09:43 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
19:09:43 INFO  Dir:   software_management_end_to_end/
19:09:43 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
19:09:43 INFO  ==============================================================
19:09:43 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:09:43 INFO  EnvironmentC8y Setup
19:09:43 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:09:44 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:09:44 INFO  Delaying execution by 6 seconds
19:10:04 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:10:04 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:10:05 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:10:05 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:10:07 INFO  State of current operation: SUCCESSFUL
19:10:08 INFO  Assert that (False == value) with value=False ... passed
19:10:08 INFO  Assert that (False == value) with value=False ... passed
19:10:08 INFO  Assert that (False == value) with value=False ... passed
19:10:08 INFO  Assert that (False == value) with value=False ... passed
19:10:08 INFO  Assert that (False == value) with value=False ... passed
19:10:10 INFO  Response status: 201
19:10:10 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-03T19:10:10.156Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-03T19:10:10.143Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334277",
    "id": "18334277",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:10:10 INFO  Started operation: {'delivery': {'time': '2022-05-03T19:10:10.156Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-03T19:10:10.143Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334277', 'id': '18334277', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
19:10:12 INFO  State of operation 18334277 : PENDING
19:10:14 INFO  State of operation 18334277 : EXECUTING
19:10:16 INFO  State of operation 18334277 : EXECUTING
19:10:19 INFO  State of operation 18334277 : EXECUTING
19:10:21 INFO  State of operation 18334277 : EXECUTING
19:10:23 INFO  State of operation 18334277 : EXECUTING
19:10:26 INFO  State of operation 18334277 : EXECUTING
19:10:28 INFO  State of operation 18334277 : EXECUTING
19:10:31 INFO  State of operation 18334277 : EXECUTING
19:10:33 INFO  State of operation 18334277 : EXECUTING
19:10:36 INFO  State of operation 18334277 : EXECUTING
19:10:38 INFO  State of operation 18334277 : EXECUTING
19:10:40 INFO  State of operation 18334277 : EXECUTING
19:10:42 INFO  State of operation 18334277 : EXECUTING
19:10:44 INFO  State of operation 18334277 : EXECUTING
19:10:47 INFO  State of operation 18334277 : EXECUTING
19:10:49 INFO  State of operation 18334277 : EXECUTING
19:10:52 INFO  State of operation 18334277 : EXECUTING
19:10:54 INFO  State of operation 18334277 : EXECUTING
19:10:57 INFO  State of operation 18334277 : EXECUTING
19:10:59 INFO  State of operation 18334277 : EXECUTING
19:11:02 INFO  State of operation 18334277 : EXECUTING
19:11:04 INFO  State of operation 18334277 : EXECUTING
19:11:06 INFO  State of operation 18334277 : EXECUTING
19:11:09 INFO  State of operation 18334277 : EXECUTING
19:11:11 INFO  State of operation 18334277 : EXECUTING
19:11:13 INFO  State of operation 18334277 : EXECUTING
19:11:15 INFO  State of operation 18334277 : EXECUTING
19:11:18 INFO  State of operation 18334277 : EXECUTING
19:11:20 INFO  State of operation 18334277 : EXECUTING
19:11:22 INFO  State of operation 18334277 : EXECUTING
19:11:24 INFO  State of operation 18334277 : EXECUTING
19:11:27 INFO  State of operation 18334277 : EXECUTING
19:11:29 INFO  State of operation 18334277 : EXECUTING
19:11:31 INFO  State of operation 18334277 : EXECUTING
19:11:33 INFO  State of operation 18334277 : EXECUTING
19:11:36 INFO  State of operation 18334277 : EXECUTING
19:11:38 INFO  State of operation 18334277 : EXECUTING
19:11:40 INFO  State of operation 18334277 : EXECUTING
19:11:43 INFO  State of operation 18334277 : EXECUTING
19:11:45 INFO  State of operation 18334277 : EXECUTING
19:11:48 INFO  State of operation 18334277 : EXECUTING
19:11:50 INFO  State of operation 18334277 : EXECUTING
19:11:52 INFO  State of operation 18334277 : EXECUTING
19:11:55 INFO  State of operation 18334277 : EXECUTING
19:11:57 INFO  State of operation 18334277 : EXECUTING
19:11:59 INFO  State of operation 18334277 : EXECUTING
19:12:01 INFO  State of operation 18334277 : EXECUTING
19:12:04 INFO  State of operation 18334277 : EXECUTING
19:12:06 INFO  State of operation 18334277 : EXECUTING
19:12:08 INFO  State of operation 18334277 : EXECUTING
19:12:11 INFO  State of operation 18334277 : EXECUTING
19:12:13 INFO  State of operation 18334277 : EXECUTING
19:12:16 INFO  State of operation 18334277 : EXECUTING
19:12:18 INFO  State of operation 18334277 : EXECUTING
19:12:20 INFO  State of operation 18334277 : EXECUTING
19:12:22 INFO  State of operation 18334277 : EXECUTING
19:12:25 INFO  State of operation 18334277 : EXECUTING
19:12:27 INFO  State of operation 18334277 : EXECUTING
19:12:29 INFO  State of operation 18334277 : EXECUTING
19:12:32 INFO  State of operation 18334277 : EXECUTING
19:12:34 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
19:12:34 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
19:12:34 INFO  
19:12:34 INFO  cleanup:
19:12:41 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:12:41 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:12:41 INFO  
19:12:41 INFO  Test duration: 178.24 secs
19:12:41 INFO  Test final outcome:  BLOCKED
19:12:41 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
19:12:41 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>19:31:53 INFO  ==============================================================
19:31:53 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
19:31:53 INFO  Dir:   software_management_end_to_end/
19:31:53 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
19:31:53 INFO  ==============================================================
19:31:53 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:31:53 INFO  EnvironmentC8y Setup
19:31:54 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:31:54 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
19:31:54 INFO  Delaying execution by 7 seconds
19:32:15 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:32:15 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:32:16 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:32:16 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:32:18 INFO  State of current operation: SUCCESSFUL
19:32:19 INFO  Assert that (False == value) with value=False ... passed
19:32:19 INFO  Assert that (False == value) with value=False ... passed
19:32:19 INFO  Assert that (False == value) with value=False ... passed
19:32:19 INFO  Assert that (False == value) with value=False ... passed
19:32:20 INFO  Package moon-buggy is installed
19:32:20 WARN  Assert that (False == value) with value=True ... failed 
19:32:21 INFO  Response status: 201
19:32:21 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-03T19:32:21.334Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-03T19:32:21.322Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334454",
    "id": "18334454",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
19:32:21 INFO  Started operation: {'delivery': {'time': '2022-05-03T19:32:21.334Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-03T19:32:21.322Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334454', 'id': '18334454', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
19:32:23 INFO  State of operation 18334454 : PENDING
19:32:25 INFO  State of operation 18334454 : EXECUTING
19:32:28 INFO  State of operation 18334454 : EXECUTING
19:32:30 INFO  State of operation 18334454 : EXECUTING
19:32:32 INFO  State of operation 18334454 : EXECUTING
19:32:34 INFO  State of operation 18334454 : EXECUTING
19:32:37 INFO  State of operation 18334454 : EXECUTING
19:32:39 INFO  State of operation 18334454 : EXECUTING
19:32:42 INFO  State of operation 18334454 : EXECUTING
19:32:44 INFO  State of operation 18334454 : EXECUTING
19:32:47 INFO  State of operation 18334454 : EXECUTING
19:32:49 INFO  State of operation 18334454 : EXECUTING
19:32:51 INFO  State of operation 18334454 : EXECUTING
19:32:53 INFO  State of operation 18334454 : EXECUTING
19:32:56 INFO  State of operation 18334454 : EXECUTING
19:32:58 INFO  State of operation 18334454 : EXECUTING
19:33:00 INFO  State of operation 18334454 : EXECUTING
19:33:02 INFO  State of operation 18334454 : EXECUTING
19:33:05 INFO  State of operation 18334454 : EXECUTING
19:33:08 INFO  State of operation 18334454 : EXECUTING
19:33:10 INFO  State of operation 18334454 : EXECUTING
19:33:12 INFO  State of operation 18334454 : EXECUTING
19:33:15 INFO  State of operation 18334454 : EXECUTING
19:33:17 INFO  State of operation 18334454 : EXECUTING
19:33:19 INFO  State of operation 18334454 : EXECUTING
19:33:21 INFO  State of operation 18334454 : EXECUTING
19:33:24 INFO  State of operation 18334454 : EXECUTING
19:33:26 INFO  State of operation 18334454 : EXECUTING
19:33:29 INFO  State of operation 18334454 : EXECUTING
19:33:31 INFO  State of operation 18334454 : EXECUTING
19:33:34 INFO  State of operation 18334454 : EXECUTING
19:33:36 INFO  State of operation 18334454 : EXECUTING
19:33:38 INFO  State of operation 18334454 : EXECUTING
19:33:41 INFO  State of operation 18334454 : EXECUTING
19:33:43 INFO  State of operation 18334454 : EXECUTING
19:33:45 INFO  State of operation 18334454 : EXECUTING
19:33:47 INFO  State of operation 18334454 : EXECUTING
19:33:49 INFO  State of operation 18334454 : EXECUTING
19:33:52 INFO  State of operation 18334454 : EXECUTING
19:33:55 INFO  State of operation 18334454 : EXECUTING
19:33:57 INFO  State of operation 18334454 : EXECUTING
19:33:59 INFO  State of operation 18334454 : EXECUTING
19:34:02 INFO  State of operation 18334454 : EXECUTING
19:34:04 INFO  State of operation 18334454 : EXECUTING
19:34:06 INFO  State of operation 18334454 : EXECUTING
19:34:08 INFO  State of operation 18334454 : EXECUTING
19:34:11 INFO  State of operation 18334454 : EXECUTING
19:34:13 INFO  State of operation 18334454 : EXECUTING
19:34:16 INFO  State of operation 18334454 : EXECUTING
19:34:19 INFO  State of operation 18334454 : EXECUTING
19:34:21 INFO  State of operation 18334454 : EXECUTING
19:34:23 INFO  State of operation 18334454 : EXECUTING
19:34:26 INFO  State of operation 18334454 : EXECUTING
19:34:28 INFO  State of operation 18334454 : EXECUTING
19:34:30 INFO  State of operation 18334454 : EXECUTING
19:34:33 INFO  State of operation 18334454 : EXECUTING
19:34:35 INFO  State of operation 18334454 : EXECUTING
19:34:37 INFO  State of operation 18334454 : EXECUTING
19:34:40 INFO  State of operation 18334454 : EXECUTING
19:34:42 INFO  State of operation 18334454 : EXECUTING
19:34:44 INFO  State of operation 18334454 : EXECUTING
19:34:46 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
19:34:47 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
19:34:47 INFO  
19:34:47 INFO  cleanup:
19:34:51 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:34:52 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:34:52 INFO  
19:34:52 INFO  Test duration: 178.90 secs
19:34:52 INFO  Test final outcome:  BLOCKED
19:34:52 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
19:34:52 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.08 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.03 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>19:18:51 INFO  ==============================================================
19:18:51 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
19:18:51 INFO  Dir:   software_management_end_to_end/
19:18:51 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
19:18:51 INFO  ==============================================================
19:18:53 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:18:53 INFO  EnvironmentC8y Setup
19:18:53 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:19:09 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 15 secs
19:19:09 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:19:10 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:19:10 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:19:13 INFO  State of current operation: SUCCESSFUL
19:19:13 INFO  Response status: 201
19:19:13 INFO  Response to action: {
    "creationTime": "2022-05-03T19:19:13.226Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334332",
    "id": "18334332",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
19:19:13 INFO  Started operation: {'creationTime': '2022-05-03T19:19:13.226Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334332', 'id': '18334332', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
19:19:15 INFO  State of operation 18334332 : PENDING
19:19:17 INFO  State of operation 18334332 : EXECUTING
19:19:19 INFO  State of operation 18334332 : EXECUTING
19:19:22 INFO  State of operation 18334332 : EXECUTING
19:19:24 INFO  State of operation 18334332 : FAILED
19:19:24 ERROR Stopping as the operation has failed
19:19:24 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
19:19:24 WARN  SystemError: The operation has failed ... blocked 
19:19:24 INFO  
19:19:24 INFO  cleanup:
19:19:28 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:19:29 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:19:29 INFO  
19:19:29 INFO  Test duration: 37.96 secs
19:19:29 INFO  Test final outcome:  BLOCKED
19:19:29 INFO  Test outcome reason: SystemError: The operation has failed
19:19:29 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>19:37:27 INFO  ==============================================================
19:37:27 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
19:37:27 INFO  Dir:   software_management_end_to_end/
19:37:27 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
19:37:27 INFO  ==============================================================
19:37:27 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:37:27 INFO  EnvironmentC8y Setup
19:37:28 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:37:28 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
19:37:28 INFO  Delaying execution by 6 seconds
19:37:48 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
19:37:49 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:37:49 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:37:50 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:37:52 INFO  State of current operation: SUCCESSFUL
19:37:52 INFO  Response status: 201
19:37:52 INFO  Response to action: {
    "creationTime": "2022-05-03T19:37:52.569Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334498",
    "id": "18334498",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
19:37:52 INFO  Started operation: {'creationTime': '2022-05-03T19:37:52.569Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334498', 'id': '18334498', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
19:37:54 INFO  State of operation 18334498 : PENDING
19:37:57 INFO  State of operation 18334498 : EXECUTING
19:37:59 INFO  State of operation 18334498 : EXECUTING
19:38:01 INFO  State of operation 18334498 : EXECUTING
19:38:03 INFO  State of operation 18334498 : FAILED
19:38:03 ERROR Stopping as the operation has failed
19:38:03 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
19:38:03 WARN  SystemError: The operation has failed ... blocked 
19:38:03 INFO  
19:38:03 INFO  cleanup:
19:38:07 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:38:08 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:38:08 INFO  
19:38:08 INFO  Test duration: 40.49 secs
19:38:08 INFO  Test final outcome:  BLOCKED
19:38:08 INFO  Test outcome reason: SystemError: The operation has failed
19:38:08 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_fake_install_remove
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.02 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.02 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
18:34:49 INFO  ==============================================================
18:34:49 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.02 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>19:39:44 INFO  ==============================================================
19:39:44 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
19:39:44 INFO  Dir:   software_management_end_to_end/
19:39:44 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
19:39:44 INFO  ==============================================================
19:39:44 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
19:39:44 INFO  
19:39:44 INFO  Test duration: 0.05 secs
19:39:44 INFO  Test final outcome:  SKIPPED
19:39:44 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
19:39:44 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>18:34:49 INFO  ==============================================================
18:34:49 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
18:34:49 INFO  Dir:   software_management_end_to_end/
18:34:49 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
18:34:49 INFO  ==============================================================
18:34:49 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
18:34:49 WARN  Aborted test due to SKIPPED outcome
18:34:49 INFO  
18:34:49 INFO  Test duration: 0.03 secs
18:34:49 INFO  Test final outcome:  SKIPPED
18:34:49 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
18:34:49 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>19:39:44 INFO  ==============================================================
19:39:44 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
19:39:44 INFO  Dir:   software_management_end_to_end/
19:39:44 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
19:39:44 INFO  ==============================================================
19:39:44 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
19:39:44 INFO  EnvironmentC8y Setup
19:39:44 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
19:39:45 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
19:39:45 INFO  Delaying execution by 7 seconds
19:40:05 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
19:40:06 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
19:40:07 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
19:40:07 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
19:40:09 INFO  State of current operation: SUCCESSFUL
19:40:10 INFO  Package apple is installed
19:40:10 INFO  Assert that (True == value) with value=True ... passed
19:40:10 INFO  Package banana is installed
19:40:10 INFO  Assert that (True == value) with value=True ... passed
19:40:10 INFO  Package cherry is installed
19:40:10 INFO  Assert that (True == value) with value=True ... passed
19:40:10 INFO  Package asciijump is installed
19:40:10 WARN  Assert that (False == value) with value=True ... failed 
19:40:11 INFO  Assert that (False == value) with value=False ... passed
19:40:11 INFO  Response status: 201
19:40:11 INFO  Response to action: {
    "creationTime": "2022-05-03T19:40:11.206Z",
    "deviceName": "m32sd10b",
    "deviceId": "18322240",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334536",
    "id": "18334536",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
19:40:11 INFO  Started operation: {'creationTime': '2022-05-03T19:40:11.206Z', 'deviceName': 'm32sd10b', 'deviceId': '18322240', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18334536', 'id': '18334536', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
19:40:13 INFO  State of operation 18334536 : PENDING
19:40:15 INFO  State of operation 18334536 : EXECUTING
19:40:17 INFO  State of operation 18334536 : EXECUTING
19:40:20 INFO  State of operation 18334536 : EXECUTING
19:40:22 INFO  State of operation 18334536 : EXECUTING
19:40:24 INFO  State of operation 18334536 : EXECUTING
19:40:27 INFO  State of operation 18334536 : EXECUTING
19:40:29 INFO  State of operation 18334536 : EXECUTING
19:40:32 INFO  State of operation 18334536 : EXECUTING
19:40:34 INFO  State of operation 18334536 : EXECUTING
19:40:36 INFO  State of operation 18334536 : EXECUTING
19:40:39 INFO  State of operation 18334536 : EXECUTING
19:40:41 INFO  State of operation 18334536 : EXECUTING
19:40:43 INFO  State of operation 18334536 : EXECUTING
19:40:45 INFO  State of operation 18334536 : EXECUTING
19:40:47 INFO  State of operation 18334536 : EXECUTING
19:40:50 INFO  State of operation 18334536 : FAILED
19:40:50 ERROR Stopping as the operation has failed
19:40:50 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
19:40:50 WARN  SystemError: The operation has failed ... blocked 
19:40:50 INFO  
19:40:50 INFO  cleanup:
19:40:54 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
19:40:54 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
19:40:54 INFO  
19:40:54 INFO  Test duration: 70.46 secs
19:40:54 INFO  Test final outcome:  BLOCKED
19:40:54 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
19:40:54 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session" />
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_fail_noarg
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.04 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_finalize
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin finalize - dense version
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin prepare - dense version
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install_fail
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin prepare - failing
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install_local_file
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin install from local file - SUCCESS case
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install_local_file_fail
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin install from local file - FAIL case
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.03 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install_local_file_with_version
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:06 INFO  ==============================================================
18:25:06 INFO  Id:    apt_install_local_file_with_version_fail
18:25:06 INFO  Dir:   plugin_apt/
18:25:06 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
18:25:06 INFO  ==============================================================
18:25:06 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:06 WARN  Aborted test due to SKIPPED outcome
18:25:06 INFO  
18:25:06 INFO  Test duration: 0.02 secs
18:25:06 INFO  Test final outcome:  SKIPPED
18:25:06 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:06 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:09 INFO  ==============================================================
18:25:09 INFO  Id:    apt_prepare
18:25:09 INFO  Dir:   plugin_apt/
18:25:09 INFO  Title: Validate apt plugin install - dense version
18:25:09 INFO  ==============================================================
18:25:09 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:09 WARN  Aborted test due to SKIPPED outcome
18:25:09 INFO  
18:25:09 INFO  Test duration: 0.02 secs
18:25:09 INFO  Test final outcome:  SKIPPED
18:25:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:09 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:09 INFO  ==============================================================
18:25:09 INFO  Id:    apt_prepare_install_finalize
18:25:09 INFO  Dir:   plugin_apt/
18:25:09 INFO  Title: Validate apt plugin install use case
18:25:09 INFO  ==============================================================
18:25:09 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:09 WARN  Aborted test due to SKIPPED outcome
18:25:09 INFO  
18:25:09 INFO  Test duration: 0.02 secs
18:25:09 INFO  Test final outcome:  SKIPPED
18:25:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:09 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:09 INFO  ==============================================================
18:25:09 INFO  Id:    apt_remove
18:25:09 INFO  Dir:   plugin_apt/
18:25:09 INFO  Title: Validate apt plugin prepare - dense version
18:25:09 INFO  ==============================================================
18:25:09 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:09 WARN  Aborted test due to SKIPPED outcome
18:25:09 INFO  
18:25:09 INFO  Test duration: 0.02 secs
18:25:09 INFO  Test final outcome:  SKIPPED
18:25:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:09 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:09 INFO  ==============================================================
18:25:09 INFO  Id:    apt_remove_fail
18:25:09 INFO  Dir:   plugin_apt/
18:25:09 INFO  Title: Validate apt plugin prepare - failing
18:25:09 INFO  ==============================================================
18:25:09 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:09 WARN  Aborted test due to SKIPPED outcome
18:25:09 INFO  
18:25:09 INFO  Test duration: 0.02 secs
18:25:09 INFO  Test final outcome:  SKIPPED
18:25:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:09 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_sequence_install_with_version
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin install with version use case
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_sequence_install_with_version_fail
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin install with version fails use case
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_sequence_remove
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin prepare install remove finalize use case
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_sequence_remove_with_version
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin remove with version use case
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_sequence_remove_with_version_fail
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin remove with version fails when the version not found
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_update_list
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin update-list command use case
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    apt_wrong_option
18:25:10 INFO  Dir:   plugin_apt/
18:25:10 INFO  Title: Validate apt plugin install
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval">
		<failure message="FAILED: Grep on resp_jwt.out contains &quot;\[c8y/s/dat\] 71,&quot;" type="FAILED" />
		<system-out>18:21:23 INFO  ==============================================================
18:21:23 INFO  Id:    c8y_jwt_token_retrieval
18:21:23 INFO  Dir:   cloud_c8y/
18:21:23 INFO  Title: Validate the Cumulocity JWT Token Retrieval feature
18:21:23 INFO  ==============================================================
18:21:24 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:21:38 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:21:38 INFO  Started sudo&lt;resp_jwt&gt; with process id 2120
18:21:40 INFO  Executed sudo&lt;req_jwt&gt;, exit status 0
18:21:42 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:21:42 WARN  Grep on resp_jwt.out contains "\[c8y/s/dat\] 71," ... failed [run.py:60]
18:21:42 INFO  
18:21:42 INFO  cleanup:
18:21:46 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:21:46 INFO  
18:21:46 INFO  Test duration: 23.22 secs
18:21:46 INFO  Test final outcome:  FAILED
18:21:46 INFO  Test outcome reason: Grep on resp_jwt.out contains "\[c8y/s/dat\] 71,"
18:21:46 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_finalize
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin finalize command
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.05 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize" />
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_install
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin install
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install" />
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_install_with_version
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin install
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version" />
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_remove
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin remove
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>19:18:36 INFO  ==============================================================
19:18:36 INFO  Id:    docker_plugin_remove
19:18:36 INFO  Dir:   plugin_docker/
19:18:36 INFO  Title: Validate docker plugin remove
19:18:36 INFO  ==============================================================
19:18:37 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:18:37 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
19:18:39 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
19:18:39 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

19:18:39 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
19:18:39 WARN  Aborted test due to BLOCKED outcome
19:18:39 INFO  
19:18:39 INFO  cleanup:
19:18:39 INFO  Removing containers scheduled to be cleaned up: ['4ea2afc6d535', '96d25fc67e59']
19:18:40 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:18:40 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
19:18:41 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:18:41 INFO  
19:18:41 INFO  Test duration: 5.03 secs
19:18:41 INFO  Test final outcome:  BLOCKED
19:18:41 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
19:18:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_remove_with_version
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin remove with version
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>19:18:41 INFO  ==============================================================
19:18:41 INFO  Id:    docker_plugin_remove_with_version
19:18:41 INFO  Dir:   plugin_docker/
19:18:41 INFO  Title: Validate docker plugin remove with version
19:18:41 INFO  ==============================================================
19:18:42 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:18:42 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
19:18:44 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
19:18:44 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

19:18:44 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
19:18:44 WARN  Aborted test due to BLOCKED outcome
19:18:44 INFO  
19:18:44 INFO  cleanup:
19:18:44 INFO  Removing containers scheduled to be cleaned up: ['4ea2afc6d535', '96d25fc67e59']
19:18:45 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:18:45 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
19:18:46 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:18:46 INFO  
19:18:46 INFO  Test duration: 4.70 secs
19:18:46 INFO  Test final outcome:  BLOCKED
19:18:46 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
19:18:46 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>18:25:10 INFO  ==============================================================
18:25:10 INFO  Id:    docker_plugin_update
18:25:10 INFO  Dir:   plugin_docker/
18:25:10 INFO  Title: Validate docker plugin install updates existing containers
18:25:10 INFO  ==============================================================
18:25:10 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
18:25:10 WARN  Aborted test due to SKIPPED outcome
18:25:10 INFO  
18:25:10 INFO  Test duration: 0.02 secs
18:25:10 INFO  Test final outcome:  SKIPPED
18:25:10 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
18:25:10 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>19:18:46 INFO  ==============================================================
19:18:46 INFO  Id:    docker_plugin_update
19:18:46 INFO  Dir:   plugin_docker/
19:18:46 INFO  Title: Validate docker plugin install updates existing containers
19:18:46 INFO  ==============================================================
19:18:47 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
19:18:47 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
19:18:49 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
19:18:49 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

19:18:49 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
19:18:49 WARN  Aborted test due to BLOCKED outcome
19:18:49 INFO  
19:18:49 INFO  cleanup:
19:18:49 INFO  Removing containers scheduled to be cleaned up: ['4ea2afc6d535', '96d25fc67e59']
19:18:50 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
19:18:50 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
19:18:51 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
19:18:51 INFO  
19:18:51 INFO  Test duration: 4.79 secs
19:18:51 INFO  Test final outcome:  BLOCKED
19:18:51 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
19:18:51 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>18:22:06 INFO  ==============================================================
18:22:06 INFO  Id:    log_request_error_messages
18:22:06 INFO  Dir:   log_request_end_to_end/
18:22:06 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
18:22:06 INFO  ==============================================================
18:22:08 INFO  EnvironmentC8y Setup
18:22:08 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
18:22:08 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
18:22:08 INFO  Delaying execution by 8 seconds
18:22:31 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:22:31 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
18:22:32 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
18:22:32 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
18:22:33 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:22:33 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
18:22:34 INFO  log response None
18:22:34 WARN  retry, retrying in 1 seconds...
18:22:35 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18323238/binaries
18:22:36 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
18:22:36 INFO  
18:22:36 INFO  cleanup:
18:22:36 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:22:37 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
18:22:38 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
18:22:42 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:22:42 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
18:22:42 INFO  
18:22:42 INFO  Test duration: 36.15 secs
18:22:42 INFO  Test final outcome:  FAILED
18:22:42 INFO  Test outcome reason: Assert that (True == value) with value=False
18:22:42 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>18:22:42 INFO  ==============================================================
18:22:42 INFO  Id:    log_request_num_lines
18:22:42 INFO  Dir:   log_request_end_to_end/
18:22:42 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
18:22:42 INFO  ==============================================================
18:22:42 INFO  EnvironmentC8y Setup
18:22:43 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
18:22:43 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
18:22:43 INFO  Delaying execution by 7 seconds
18:23:04 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
18:23:05 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
18:23:05 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
18:23:06 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
18:23:06 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:23:07 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
18:23:08 INFO  log response None
18:23:08 WARN  retry, retrying in 1 seconds...
18:23:09 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18324476/binaries
18:23:09 INFO  nlines 0
18:23:09 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
18:23:09 INFO  
18:23:09 INFO  cleanup:
18:23:10 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
18:23:10 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
18:23:11 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
18:23:15 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
18:23:15 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
18:23:15 INFO  
18:23:15 INFO  Test duration: 33.02 secs
18:23:15 INFO  Test final outcome:  FAILED
18:23:15 INFO  Test outcome reason: Assert that (True == value) with value=False
18:23:15 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>18:37:51 INFO  ==============================================================
18:37:51 INFO  Id:    mapper_awaits_before_reconnect
18:37:51 INFO  Dir:   tedge_mapper_c8y/
18:37:51 INFO  Title: Validate mapper doesn't reconnect too often
18:37:51 INFO  ==============================================================
18:37:51 INFO  Started mosquitto_sub with process id 7103
18:37:51 INFO  Waiting for 0.1 seconds
18:37:52 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
18:37:52 INFO  Waiting for 5.0 seconds
18:37:57 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
18:37:58 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
18:37:58 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
18:37:58 INFO  Line count on file tedge_mapper_journal.out ... passed
18:37:58 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
18:37:58 WARN  Aborted test due to FAILED outcome
18:37:58 INFO  
18:37:58 INFO  Test duration: 7.49 secs
18:37:58 INFO  Test final outcome:  FAILED
18:37:58 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
18:37:58 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>18:24:44 INFO  ==============================================================
18:24:44 INFO  Id:    monitoring_smaller_interval
18:24:44 INFO  Dir:   monitoring/
18:24:44 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
18:24:44 INFO  ==============================================================
18:24:46 INFO  Executed sudo&lt;collectd&gt;, exit status 0
18:24:47 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:24:47 INFO  Started sudo&lt;tedge_sub&gt; with process id 3037
18:24:47 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:48 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:49 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:49 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:50 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:50 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:51 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:52 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:52 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:53 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:53 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:54 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:54 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:55 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:56 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:56 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:57 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:57 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:24:58 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:24:58 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:25:00 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:25:00 INFO  0

18:25:00 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
18:25:00 WARN  TypeError: 'int' object is not subscriptable ... blocked 
18:25:00 INFO  
18:25:00 INFO  cleanup:
18:25:00 INFO  monitoring_cleanup
18:25:01 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:25:01 INFO  
18:25:01 INFO  Test duration: 16.33 secs
18:25:01 INFO  Test final outcome:  BLOCKED
18:25:01 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
18:25:01 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>18:25:01 INFO  ==============================================================
18:25:01 INFO  Id:    monitoring_with_collectd
18:25:01 INFO  Dir:   monitoring/
18:25:01 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
18:25:01 INFO  ==============================================================
18:25:01 INFO  Test execution disabled due to #792 (instable) ... skipped
18:25:01 INFO  
18:25:01 INFO  Test duration: 0.02 secs
18:25:01 INFO  Test final outcome:  SKIPPED
18:25:01 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
18:25:01 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>18:25:01 INFO  ==============================================================
18:25:01 INFO  Id:    monitoring_with_simulated_messages
18:25:01 INFO  Dir:   monitoring/
18:25:01 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
18:25:01 INFO  ==============================================================
18:25:01 INFO  Executed sudo&lt;collectd&gt;, exit status 0
18:25:02 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:25:02 INFO  Started sudo&lt;tedge_sub&gt; with process id 3089
18:25:03 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
18:25:03 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
18:25:05 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:25:05 INFO  0

18:25:05 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
18:25:05 WARN  TypeError: 'int' object is not subscriptable ... blocked 
18:25:05 INFO  
18:25:05 INFO  cleanup:
18:25:05 INFO  monitoring_cleanup
18:25:05 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
18:25:05 INFO  
18:25:05 INFO  Test duration: 4.51 secs
18:25:05 INFO  Test final outcome:  BLOCKED
18:25:05 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
18:25:05 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>18:23:51 INFO  ==============================================================
18:23:51 INFO  Id:    mqtt_port_change_connection_works
18:23:51 INFO  Dir:   misc_features/
18:23:51 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
18:23:51 INFO  ==============================================================
18:23:51 INFO  failing intermitently ... skipped
18:23:51 INFO  
18:23:51 INFO  Test duration: 0.02 secs
18:23:51 INFO  Test final outcome:  SKIPPED
18:23:51 INFO  Test outcome reason: failing intermitently
18:23:51 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers">
		<failure message="FAILED: Grep on tedge_mapper2.err contains &quot;Error: Couldn't acquire file lock.&quot;" type="FAILED" />
		<system-out>18:37:59 INFO  ==============================================================
18:37:59 INFO  Id:    runtime_start_two_mappers
18:37:59 INFO  Dir:   tedge_mapper_c8y/
18:37:59 INFO  Title: Validate two mappers can't be started at the same time
18:37:59 INFO  ==============================================================
18:37:59 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
18:37:59 INFO  Waiting for 0.1 seconds
18:38:00 INFO  Executed sudo&lt;tedge_mapper2&gt;, exit status 1
18:38:00 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
18:38:00 WARN  Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock." ... failed [run.py:46]
18:38:00 INFO  
18:38:00 INFO  Test duration: 1.78 secs
18:38:00 INFO  Test final outcome:  FAILED
18:38:00 INFO  Test outcome reason: Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock."
18:38:00 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>18:35:15 INFO  ==============================================================
18:35:15 INFO  Id:    tedge_agent_user_sudo_access
18:35:15 INFO  Dir:   tedge/
18:35:15 INFO  Title: Validate tedge-agent user has a limited sudo right
18:35:15 INFO  ==============================================================
18:35:15 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
18:35:15 INFO  
18:35:15 INFO  Test duration: 0.02 secs
18:35:15 INFO  Test final outcome:  SKIPPED
18:35:15 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
18:35:15 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>18:39:36 INFO  ==============================================================
18:39:36 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
18:39:36 INFO  Dir:   tedge_mapper_c8y/
18:39:36 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
18:39:36 INFO  ==============================================================
18:39:36 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
18:39:36 INFO  Started sudo&lt;tedge_sub&gt; with process id 7691
18:39:37 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
18:39:38 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
18:39:38 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
18:39:38 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
18:39:38 INFO  
18:39:38 INFO  cleanup:
18:39:38 INFO  mapper_cleanup
18:39:39 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
18:39:39 INFO  
18:39:39 INFO  Test duration: 2.95 secs
18:39:39 INFO  Test final outcome:  FAILED
18:39:39 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
18:39:39 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>